config/initializers/assets.rb

```diff
@@ -8,7 +8,7 @@
 
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in app/assets folder are already added.
-Rails.application.config.assets.precompile += %w(misc/*.js email.css ie*.css)
+Rails.application.config.assets.precompile += %w(app/controllers/*.js misc/*.js email.css ie*.css)
```

Comment:
@el-mapache in the past, we have selectively loaded js that didn't need to be included site wide by just including it on the specific view.  The password strength meter for example: https://github.com/18F/identity-idp/blob/master/app/views/devise/passwords/edit.html.slim#L16